tools/chplcheck/src/chplcheck.py

```diff
@@ -232,6 +252,10 @@ def main():
         help="Apply fixits interactively, requires --fixit",
     )
     args = parser.parse_args()
+    args.filenames.extend(args.file)
+    args.filenames = itertools.chain(
+        *map(functools.partial(glob.glob, recursive=True), args.filenames)
+    )
```

Comment:
the point of this is to enable doing like, `files = ["test/*"]` from `Mason.toml` or something?

Reply:
That's exactly it